src/compas_viewer/scene/tagobject.py

```diff
@@ -118,6 +118,7 @@ def make_buffers(self):
         }
 
     def make_text_texture(self):
+        print(self.geometry.font)
```

Comment:
?

Reply:
is this the hotfix?

Reply:
Ahh no there was a missing file, I will remove this print line